### ISSUE TITLE
Add e2e test to ensure infinite blur/focus loop does not occur

### ISSUE THREAD

**Issue by cvializ** [ampproject/amphtml#22844]
Issue to track followup effort for https://github.com/ampproject/amphtml/pull/22838#discussion_r293973922

Make an e2e test for the date picker mobile behavior

**Comment by estherkim** [comment]
Hey @nainar, can someone from @ampproject/wg-ui-and-a11y pick this up? Happy to help if the e2e testing framework is missing API support.

**Comment by caroqliu** [comment]
I can look into this 👍
